ThirdPartyAdapters/chartboost/chartboost/src/main/java/com/google/ads/mediation/chartboost/ChartboostSingleton.java

```diff
@@ -293,6 +367,15 @@ protected static void showRewardedVideoAd(AbstractChartboostAdapterDelegate dele
         Chartboost.showRewardedVideo(delegate.getChartboostParams().getLocation());
     }
 
+    protected static void loadBannerAd(AbstractChartboostAdapterDelegate delegate) {
```

Comment:
Is this ever called?

Reply:
Needs to be called somewhere as result of a `requestBannerAd(Context, MediationBannerListener, Bundle, AdSize, MediationAdRequest, Bundle)` call.

Reply:
yes it is called in ChartboostAdapter when sdk initialises. didInitialize()

Reply:
Oh right, sorry I missed that